src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyCost = 40;
+  const weekDisc = 50;
+  const threeDDisc = 20;
+  let rentalCost = 0;
```

Comment:
```suggestion
  let rentalCost = days * dailyCost;
```
You're using this expression a lot